src/types/orderOption.graphql

```diff
@@ -0,0 +1,57 @@
+type OrderOption {
+  "The list of concepts and any data on the relationship between this order option and other permitted concepts"
+  associationDetails: JSON
+  "The concept id created by CMR for this orderOption"
+  conceptId: String!
+  "The description for this order option"
+  description: String
+  "The ECHO Forms schema for this order option"
+  form: String
+  "The guid-id for this order option"
+  id: String!
+  "The orderOption name."
+  name: String
+  "The native id to set on the order option."  
+  nativeId: String
```

Comment:
I am going to leave it in for now. I think this is an excellent point but, I do think we ought to keep things somewhat consistent. Since the `native-id` field is returned in all the other concepts and, at least as I understood from some discussion last week, one client uses `native-id `in their search (cmr-stac) for some concepts. We should consider whether to remove this field more broadly though